src/api/users/src/services/firestore.js

```diff
@@ -42,13 +42,14 @@ if (
       client_x509_cert_url: FIREBASE_CLIENT_X509_CERT_URL,
     }),
   });
-  logger.debug('Server running in online mode');
```

Comment:
I think debug is fine for these.

Reply:
I can change it back to debug but there's no notice that the express server is running otherwise. Is this a log level setting issue?

Reply:
In production we log all HTTP requests.